masterlist.yaml

```
@@ -8741,6 +8741,53 @@ plugins:
     after: [ 'SkyTEST-RealisticAnimals&Predators.esp' ]
     tag: [ Text ]
 
+  - name: 'Apothecary.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/52130/' ]
+    after: [ 'TechnicolorAlchemy.esp' ]
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Beyond Skyrim - Bruma' ]
+        condition: 'active("BSAssets.esm") or active("BSHeartland.esm") and not active("Apothecary - Bruma Patch.esp")'
```

Since the patch has both plugins as dependencies, an `and` would be more appropriate:

```suggestion
        condition: 'active("BSAssets.esm") and active("BSHeartland.esm") and not active("Apothecary - Bruma Patch.esp")'
```

---

I would remove the check for `active("BSAssets.esm")`, it's redundant.